pandas/tseries/tdi.py

```diff
@@ -440,21 +439,18 @@ def astype(self, dtype):
 
         if dtype == np.object_:
             return self.asobject
-        elif dtype == _INT64_DTYPE:
-            return self.asi8.copy()
         elif dtype == _TD_DTYPE:
             return self
         elif dtype.kind == 'm':
```

Comment:
going to anything other than ``m8[ns]`` really should be an error (or maybe return the numpy array_; but I think just raise for now.